src/libstd/sys/unix/ext.rs

```diff
@@ -121,20 +128,60 @@ impl OsStringExt for OsString {
     }
 }
 
-// Unix-specific extensions to `OsStr`.
+/// Unix-specific extensions to `OsStr`.
 pub trait OsStrExt {
+    /// Get the underlying byte view of the `OsStr` slice.
     fn as_byte_slice(&self) -> &[u8];
+
+    /// Convert the `OsStr` slice into a `CString`.
+    fn to_cstring(&self) -> CString;
```

Comment:
Oh just in the sense for an underscore between `c` and `string` because for `OsString` it was underscore'd as `os_string`